layouts/shortcodes/page/objectives.html

```diff
@@ -31,34 +31,7 @@
     </div>
   </div>
 </section>
-<section class="row text-center padding-top-20">
-  <div class="container">
-    <div class="col-xs-24 col-sm-6 match-height-item-by-row">
-      <figure class="eclipsefdn-figure padding-20">
-          <img class="img img-responsive margin-auto" src="/images/traffic_panel.jpg" alt="Overhead view of busy intersection"> 
-      </figure>
-      <p>Traffic simulation of highway, rural and urban scenarios</p> 
-    </div>
-    <div class="col-xs-24 col-sm-6 match-height-item-by-row">
-      <p>Stochastic variation of scenarios</p>
-      <figure class="eclipsefdn-figure img responsive-img margin-auto">
-          <img class="img img-responsive margin-auto" src="/images/variation_panel.jpg" alt="Overhead view of busy intersection"> 
-      </figure>
-    </div>
-    <div class="col-xs-24 col-sm-6 match-height-item-by-row">
-      <figure class="eclipsefdn-figure padding-20">
-          <img class="img img-responsive margin-auto" src="/images/interfaces_panel.jpg" alt="Man interacting with cyclic floating interface"> 
-      </figure>
-      <p>Standardized interfaces for model integration</p>
-    </div>
-    <div class="col-xs-24 col-sm-6 match-height-item-by-row" >
-      <p>Reproducibility through deterministic simulation</p>
-      <figure class="eclipsefdn-figure padding-20">
-          <img class="img img-responsive margin-auto" src="/images/reproducability_panel.jpg" alt="Man interacting with cyclic floating interface"> 
-      </figure>
-    </div>
-  </div>
-</section>
+{{< figure class="padding-30 margin-right-50 margin-left-50" src="/layouts/shortcodes/page/target-objectives.jpg" alt="target objectives" >}}
```

Comment:
As discussed in earlier comment, including a shortcode here will break the build. This could be replaced by one of the figure elements deleted above with some slight tweaking!

Reply:
Can you help us out in this case? We do not know how to tweak the desired image into the code?

Reply:
You should be able to replace:
```
{{< figure class="padding-30 margin-right-50 margin-left-50" src="/layouts/shortcodes/page/target-objectives.jpg" alt="target objectives" >}}
```
With:  
```
      <figure class="eclipsefdn-figure padding-30 margin-right-50 margin-left-50">
          <img class="img img-responsive margin-auto" src="/images/target-objectives.jpg" alt="Target objectives"> 
      </figure>
```
We should preview this to make sure it matches your intended output, but it should match. If not, it should be pretty darn close!

PLEASE NOTE: files in the layout folder are not present on the live server. The image referenced here should be moved to `/static/images/` to match this HTML, or optionally can be included in the same folder as the target-objectives page file, and update the src attribute to be `target-objectives.jpg`

Reply:
Keep in mind this is how that image now looks on mobile phones and such:

![image](https://user-images.githubusercontent.com/23018241/87955983-740d3700-ca7c-11ea-9788-ff2388484fe3.png)

Reply:
What do you suggest to get that image aligned with the grey and blue boxes? @autumnfound